.releaserc

```diff
@@ -0,0 +1,11 @@
+---
+branches:
+- master
+- name: 2021-06-release
```

Comment:
yes, we need to be hardcoded here 🤷🏼 believe me or not, this is also strange for me but `semantic-release` package doesn't support regex here (I tried everything). I will make sure this step is part of the release process procedure